Snabble/UI/Onboarding/Views/OnboardingButtonView.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  OnboardingButtonView.swift
+//  Snabble
+//
+//  Created by Uwe Tilemann on 06.08.22.
+//
+//  Copyright © 2022 snabble. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+
+/// Tinted button used for Onboarding navigation
+struct TintButton: ButtonStyle {
```

Comment:
Ist es so in Swift? Instinktiv hätte ich es `TintButtonStyle` genannt, aber kenne mich mit SwiftUI nicht so gut aus.

Reply:
d292c64b3cdbf4f50e2904f82c61097b14e17665